config/locales/models/assessments/en.yml

```diff
@@ -4,6 +4,12 @@ en:
     attributes:
       assessment:
         max_mark: Out Of
+    errors:
+      models:
+        assessment:
+          attributes:
+            short_identifier:
+              format: short_identifier must only contain alphanumeric, hyphen, or underscore
```

Comment:
Instead of hard-coding the attribute name, you should be able to use `%{attribute}`: see https://guides.rubyonrails.org/i18n.html#error-message-interpolation.

This is true for all of the validation messages.